src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ball.setColor(new ColorSupplier().getRandomColor().toString());
+        ball.setNumber(new Random().nextInt(100));
```

Comment:
create a constant from 100

Reply:
I didn't understand it "create a constant from 100". Can you explain in more detail?

Reply:
You need to create consant field for number 100 and use it here. How to create constant - https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names